system/lambdaLoadTest/playlist.xml

```diff
@@ -56,48 +54,7 @@
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
-		<platformRequirements>^os.linux</platformRequirements>
 	</test>
-	<test>
-		<testCaseName>LambdaLoadTest_J9_Linux_CompRef_5m</testCaseName>
-		<variations>
-			<variation>Mode110</variation>
-			<variation>Mode610</variation>
-		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=LambdaLoadTest -test-args=$(Q)timeLimit=5m$(Q); \
-	$(TEST_STATUS)</command>
-		<levels>
-			<level>sanity</level>
-		</levels>
-		<groups>
-			<group>system</group>
-		</groups>
-		<impls>
-			<impl>openj9</impl>
-			<impl>ibm</impl>
-		</impls>
-		<platformRequirements>os.linux,vm.cmprssptrs</platformRequirements>
-	</test>
-	<test>
-		<testCaseName>JdiTest</testCaseName>
```

Comment:
@lumpfish - Was JdiTest removed by mistake?

Reply:
Not by mistake - it does not belong in a lambda test playlist as it tests the java debug interface.  There isn't a 'debugging' playlist, so 'other' looks like the best fit. I'll create a follow on PR.